.github/workflows/ci.yaml

```diff
@@ -0,0 +1,117 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'main'
+
+env:
+    SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+jobs:
+    composer-validate:
+        name: "Composer validate (${{ matrix.php-version }})"
+
+        runs-on: ubuntu-16.04
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.4'
+
+        steps:
+            -
+                name: "Checkout code"
+                uses: "actions/checkout@v2"
+
+            -
+                name: "Install PHP"
+                uses: "shivammathur/setup-php@v2"
+                with:
+                    coverage: "none"
+                    php-version: "${{ matrix.php-version }}"
+                    tools: "composer:v2"
+
+            -
+                name: "Validate composer.json"
+                run: "composer validate --strict --no-check-lock"
+
+            -
+                name: "Composer install"
```

Comment:
why running a composer install here ?

Reply:
its just the name, internally it uses update if `highest` or `lowest` and install if `locked`, see:

https://github.com/ramsey/composer-install#dependency-versions

I removed the name of the step like down below

Reply:
but there is no point running composer at all in this step, as you never use the dependencies.

Reply:
But how can we use phpunit bridge then? 🙄

Reply:
this comment was on a separate job than the one running test. You moved the validation to the test job after that